src/build.ts

```diff
@@ -100,7 +100,8 @@ export async function writeTypes (nitro: Nitro) {
           : {}
       },
       include: [
-        './nitro.d.ts'
+        './nitro.d.ts',
+        '../../**/*'
```

Comment:
This is right, but it should be based on the relative path of the buildDir to the root directory rather than hard-coded. See https://github.com/nuxt/framework/blob/2d202b59e33e77174102cc0f6114abd700874f37/packages/nuxi/src/utils/prepare.ts#L27-L31 for example.

Reply:
ah yes, thanks for pointing this out! 👍

Reply:
Resolved.